File: EjemploJavaPython/CalcularSuma10Numeros.py
contador=1
suma_numeros=0
#nombre="David"
#Datos de Entrada y Proceso
while contador<=10:
  valor_numero=float(input(f"Ingrese el valor numerico de la posicion {contador}:"))
  suma_numeros=suma_numeros+valor_numero
  contador=contador+1
#Datos de Salida
print(f"La suma de los 10 números ingresados es:{suma_numeros}")

#print("Prueba:",suma_numeros,"-",contador,"-",nombre)
#print("Prueba:"+nombre+ " Mamani")

File: EjemploJavaPython/CalcularSuma10Numeros.cpp
#include <iostream>

void suma10Valores(){
  int contador=1;
  float suma_numeros=0, valor_numero;
  //Datos de Entrada y Proceso
  while (contador<=10){
      printf("Ingrese el valor numerico de la posicion %i:",contador);
      scanf("%f",&valor_numero);
        suma_numeros=suma_numeros+valor_numero;
      contador++; //contador=contador+1 //contador+=1
  }
  printf("La suma de los 10 números ingresados es:%.2lf",suma_numeros);
}

int main(){    
    printf("Hola mundo...\n");
    suma10Valores();
    return 0;
}